### ISSUE TITLE
Support for G2 & G3

### ISSUE THREAD

**Issue by arifbalik** [AndyGlx/FullControl-GCode-Designer#9]
I've been doing some testing and realized that circle/arc moves are implemented with G1 command. For example in the Lattice example my printer can not reach a fluid motion while doing the arcs, maybe introducing G2 & G3 will solve the problem?

**Comment by AndyGlx** [comment]
Yes this is a limitation currently. It would be possible to edit the VBA to output a different GCode style and include G2 and similar. But I'm working on a python version that will allow such modifications much much more easily

**Comment by SARankDirector-Minecraft** [comment]
YAY!
